json/innsyn/soker/parts/avsender.json

```diff
@@ -0,0 +1,21 @@
+{
+	"$schema": "http://json-schema.org/draft-06/schema#",
+	
+	"title": "Avsender",
+	"javaType": "no.nav.sbl.soknadsosialhjelp.innsyn.soker.Avsender",
+	
+	"type": "object",
+	"properties": {
+		"systemnavn": {
+			"type": "string",
+			"description": "Navnet på avsendersystemet. Brukes ved eventuell feilsøking av ugyldige data."
+		},
+		"systemversjon": {
+			"type": "string",
+			"description": "Navnet på avsendersystemet. Brukes ved eventuell feilsøking av ugyldige data."
```

Comment:
Skal kanskje stå versjon i stedet for navn..?

Reply:
Takk, det skal det stå. Retter nå :-)